core/designsystem/src/commonMain/kotlin/com/mifos/core/designsystem/component/MifosBottomSheet.kt

```diff
@@ -54,7 +55,12 @@ fun MifosBottomSheet(
     content: @Composable () -> Unit,
 ) {
     val coroutineScope = rememberCoroutineScope()
-    val modalSheetState = rememberModalBottomSheetState(skipPartiallyExpanded = true)
+    val modalSheetState = rememberModalBottomSheetState(
+        skipPartiallyExpanded = true,
+        confirmValueChange = { newValue ->
+            newValue != SheetValue.Hidden
+        },
```

Comment:
The `confirmValueChange` callback blocks the sheet from reaching `Hidden`, preventing swipe-down dismissal and breaking standard Material Design UX.

Suggestion: Remove `confirmValueChange` or allow dismissal.

```suggestion
        confirmValueChange = { newValue ->
            true  // Allow all state changes
        },
```

Reply:
I intentionally added that to prevent the Bottom Sheet from accidentally dismissing when scrolling through view charges, as that too seems like a bad UX.

Reply:
This is a screen Recording highlighting the issue i m trying to mention: 

[Screen_recording_20251008_214517.webm](https://github.com/user-attachments/assets/f97de56a-cd33-48ec-8a8f-366fe246bbaf)